raiden/tests/integration/rpc/test_assumptions.py

```diff
@@ -157,11 +159,11 @@ def test_duplicated_transaction_different_gas_price_raises(deploy_client):
         contract_proxy.contract_address,
     )
 
-    gas = contract_proxy.estimate_gas('ret') * 2
+    startgas = contract_proxy.estimate_gas('ret') * 2
```

Comment:
I don't understand the need for doubling the estimate here and below. Either don't double it, or use the safety `GAS_FACTOR` from `raiden.constants`, if this is what you want to test, instead.

Reply:
This one is suppose to send the same transaction but with a different `startgas` and the same `gasprice`, the test is `test_duplicated_transaction_different_gas_price_raises`

Reply:
IIRC this is to test decreasing gas prices for two transactions, therefore this `make_decreasing_gas_price_strategy` factory is used. The gas limit is not important here, but I fixed it to use `safe_gas_limit` as well.